tutorials/dataframe/df021_createTGraph.C

```diff
@@ -0,0 +1,33 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// \notebook
+/// This tutorial shows how to fill a TGraph using the Dataframe.
+///
+/// \macro_code
+///
+/// \date July 2018
+/// \author Enrico Guiraud, Danilo Piparo, Massimo Tumolo
+
+
+
+void df021_createTGraph()
+{
+   std::vector<int> source(160);
+   for (int i = 0; i < 160; ++i)
```

Comment:
Maybe we can replace this with `std::iota` https://en.cppreference.com/w/cpp/algorithm/iota ?